assets/css/all.sass

```
@@ -47,12 +47,12 @@ body
 // Dark mode using media query
 @media (prefers-color-scheme: dark)
   .site-container
```

If you add this, the background will also be properly coloured when the user scrolls past the end/beginning of a webpage:

```suggestion
  html, body
    background-color: #1D1E20

  .site-container
```